examples/library/src/Server.hs

```diff
@@ -1,12 +1,9 @@
 {-# language DataKinds             #-}
-{-# language FlexibleContexts      #-}
 {-# language OverloadedStrings     #-}
 {-# language PartialTypeSignatures #-}
-{-# language ScopedTypeVariables   #-}
 {-# language TypeApplications      #-}
-{-# language TypeFamilies          #-}
 {-# language TypeOperators         #-}
-{-# OPTIONS_GHC -fno-warn-partial-type-signatures #-}
+{-# options_ghc -fno-warn-partial-type-signatures #-}
```

Comment:
Can you actually do this? I thought that had to be in uppercase.

Reply:
It works 🤷🏻‍♂️  😆